Makefile

```
@@ -41,14 +41,16 @@ temp-pc:
 	${SED} "s#/include/libgovarnam##g" govarnam.pc
 	${SED} "s#/lib\$$##g" govarnam.pc
 
+.PHONY: install-script
 install-script:
```

```suggestion
install.sh: install.sh.in
```

This would make int non phony and only rebuild when `install.sh.in` changes.